.storybook/custom/components/DocsPage.js

```diff
@@ -41,7 +41,17 @@ const DocsPage = () => {
     let [themeState, setThemeState] = useState('sap_fiori_3');
     const previousTheme = useRef();
 
+    let [directionalityState, setDirectionalityState] = useState('ltr');
+    const previousDirectionality = useRef();
+
     useEffect(() => {
+        if(!previousDirectionality.current || previousDirectionality.current !== directionalityState){
+            document.getElementsByClassName('sbdocs-preview').forEach(item => {
+                item.childNodes[1].setAttribute('dir', directionalityState);
+            });
+            previousDirectionality.current = directionalityState;
+        }
+        
```

Comment:
While you're here, would you mind removing the `console.log(item)` I accidentally left on line 50

Reply:
done

Reply:
Thank you!